File: sbt-plugin/src/sbt-test/scala-ts-sbt/simple/src/test/resources/CaseClassFoo.ts
export interface CaseClassFoo {
  id: string;
  name: string;
  i: number;
  flag: number;
  score: number;
  time: number;
  localDate: Date;
  instant: Date;
  localDateTime: Date;
  offsetDateTime: Date;
  zonedDateTime: Date;
  ts: Date;
  tuple2: [string, number];
  tuple3: [Date, Date, CaseClassFoo];
}

File: sbt-plugin/src/sbt-test/sbt-scala-ts/multi/api/src/main/scala/CaseClassFoo.scala
package io.github.scalats.sbttest

import java.time._

case class CaseClassFoo(
    id: java.util.UUID,
    name: String,
    i: Int,
    flag: Byte,
    score: Short,
    time: Long,
    localDate: LocalDate,
    instant: Instant,
    localDateTime: LocalDateTime,
    offsetDateTime: OffsetDateTime,
    zonedDateTime: ZonedDateTime,
    ts: java.sql.Timestamp,
    tuple2: (String, Int),
    tuple3: (Instant, LocalDate, CaseClassFoo),
    bar: Option[CaseClassBar])
